### PR TITLE
Collar Me Surprised

### PR DESCRIPTION
**Pull request by Memeji**
# Description

This update adds some collars to the game. Many found in the various department wardrobe vendors.

---

# Changelog

:cl:
- add: Black Collar to Clothesmate and Loadout
- add: Blue Collar Clothesmate and Loadout
- add: CentCom Collar to CentDrobe
- add: Captain's Collar to the Uniform Printer and Captain's Loadout
- add: Engineer Collar to EngiDrobe and Engineer's Loadout
- add: Epistemics Collar to SciDrobe and Epistemic's Loadout
- add: Logistics Collar to CargoDrobe and Logistic's Loadout
- add: Medical Collar to MediDrobe and Medical's Loadout
- add: Security Collar to SecDrobe and Security's Loadout
- add: Blood-Red Collar to SyndieDrobe and Loadout

### CODE REVIEW

**Review by FoxxoTrystan — APPROVED** [review]

**Comment by FoxxoTrystan** [comment]
All look good, tho do you think you could add those in "Loadout" also?
in the neck category.

**Comment by Memeji** [comment]
> All look good, tho do you think you could add those in "Loadout" also? in the neck category.

Doing so now! Didn't notice it till I already submitted it so I'll quickly get them added.

File: Resources/Prototypes/Loadouts/Jobs/Heads/captain.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/Jobs/Heads/captain.yml:1]: Please use the "Floof" folder.

File: Resources/Prototypes/Loadouts/Jobs/cargo.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/Jobs/cargo.yml:1]: Please use the "Floof" folder.

File: Resources/Prototypes/Loadouts/Jobs/engineering.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/Jobs/engineering.yml:1]: Please use the "Floof" folder.

File: Resources/Prototypes/Loadouts/Jobs/medical.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/Jobs/medical.yml:1]: Please use the "Floof" folder.

File: Resources/Prototypes/Loadouts/Jobs/science.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/Jobs/science.yml:1]: Please use the "Floof" folder.

File: Resources/Prototypes/Loadouts/Jobs/security.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/Jobs/security.yml:1]: Please use the "Floof" folder.

File: Resources/Prototypes/Loadouts/neck.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/neck.yml:1]: Please use the "Floof" folder.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

**Review by FoxxoTrystan — APPROVED** [review]

**Comment by Mnemotechnician** [comment]
This should absolutely be ported to EE

**Comment by FoxxoTrystan** [comment]
> This should absolutely be ported to EE

Okay, go fight VM to add this.

**Pull request merged**